scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatOps.scala

```diff
@@ -942,14 +949,18 @@ class FormatOps(
 
   def typeTemplateSplits(
       template: Template,
-      indent: Int
-  ): Seq[Split] = {
+      indent: Int,
+      isDerives: Boolean = false
+  )(implicit line: sourcecode.Line): Seq[Split] = {
     val hasSelfAnnotation = template.self.tokens.nonEmpty
-    val expire = (template.parent match {
+    val curlyOrDerivesOrWith = template.parent match {
       case Some(_: Defn.Given) =>
         findLast(template.tokens)(x => x.is[T.KwWith] && owners(x) == template)
-      case _ => templateCurly(template)
-    }).getOrElse(template.tokens.last)
```

Comment:
And this line also didn't need to change

Reply:
Good catch, fixed